### DISCUSSION TITLE
Get Copilot warning on github.com

### DISCUSSION THREAD

**Discussion by tshu-w** [community/community#138887] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi, I got  "Your GitHub Copilot setup is incomplete. Please go to your settings page and complete your setup." on my Github homepage and I don't know how to disable it.
<img width="1073" alt="Screenshot 2024-09-17 at 17 04 09" src="https://github.com/user-attachments/assets/e3afd137-a806-4fd0-be17-90d45ceb9726">

<img width="1117" alt="Screenshot 2024-09-17 at 17 13 29" src="https://github.com/user-attachments/assets/47d1a7c1-9568-41ed-87bc-a116d49d3422">

**Comment by gimlichael** [comment]
I have the same - surfaced a few days ago. I cannot get rid of that annoying banner (and nothing to configure).

**Comment by robswc** [comment]
Same exact thing happening to me.  Going to settings doesn't reveal anything.

**Comment by slawoe** [comment]
Same here.

I have not checked/agreed the "Allow GitHub to use my code snippets from the code editor for product improvements *".
But GitHub is not showing that it is required or reveals any mandatory steps for finishing the "incomplete" setup.

**Reply by i40west** [reply]
I've had that option enabled the entire time, and I'm suddenly getting this message too, so it's unrelated to that.

**Comment by Hentioe** [comment]
What exactly does this warning want us to do? Does it want us to check "Allow GitHub to use my code snippets from the code editor for product improvements"?

Is it related to free use?

**Comment by RaniAgus** [comment]
Same here

**Comment by Missuo0o** [comment]
The same

**Comment by aiekick** [comment]
yep its funny to see he is "Enabled" but there is no way to DISABLE this shit.. even if its free since i have a pro account... i not bought a pro account for that shit

**Comment by kevinetore** [comment]
Same issue here.

**Comment by HichemTab-tech** [comment]
Same :)

**Comment by aromanro** [comment]
Idem.

**Comment by Twelve57** [comment]
+1 👍

**Comment by mspronesti** [comment]
Same.

**Comment by RoboPsycho** [comment]
I am also getting "Sorry, your request was rate-limited. Please wait and try sending again later." inside of vscode

**Comment by aamirazad** [comment]
Same

**Comment by WinkelCode** [comment]
Also seeing this, suggestions matching public code are allowed, but code snippets for product improvement are off. Copilot works fine in VSCode, both completion and chat.

**Comment by rogergcc** [comment]
same.
It was workin nice hours ago but now not and appear this warning in githuv

update sing out copilot and sign in tehne disable and enabled plugin
wil wookrs

**Comment by davevad93** [comment, marked as answer]
Hi @tshu-w and everyone,

I've had that warning on my dashboard all day long as well; a few minutes ago it disappeared for good.

**Comment by HichemTab-tech** [comment]
yup @davevad93 me too, i think it was a bug and they fixed it.

**Comment by gimlichael** [comment]
Eventually, I experienced an error in Visual Studio 2022 (Preview) that forced me to refresh the auth-token with GitHub. It failed. I closed all VS-instances (as it was working in some of them). Shortly hereafter, the message disappeared. As others has mentioned, it was probable a bug given the magnitude of false-positives.

Cheers.

**Comment by piyushjain5703** [comment]
Hey,

Enabling the **`Suggestions matching public code (duplication detection filter)`** option worked for me. It removed the warning, and I was able to use Copilot successfully.

You can give it a try — it might work for you too!

Cheers

**Reply by kiuverkaddiel** [reply]
It work for me too. I has been suffert for days. Thanks.

**Reply by tyronelee0329** [reply]
thannnnnks. I've been struggling with the banner for days.
